docs/topics/levelstring_encoding_decoding.md

```diff
@@ -8,6 +8,8 @@ Compressed levels are actually composed out of two things: [Base64](topics/encry
 ## Encoding
 To encode a level you take first the level string and `compress()` it, afterwards you need to encode the resulting byte sequence with Base64 encoding.
 
+If you are encoding an official level and want to put it in `LevelData.plist`, you need to compress the string at only level 6, replace the first 10 bytes to `\x1f\x8b\x08\x00\x00\x00\x00\x00\x00\x03` after compressing, then remove `H4sIAAAAAAAAA` in the beginning of the base64 encoded string. (actually the game itself has hardcoded values to verify the string length, so the best way to replace levels is using a hack)
```

Comment:
`(actually the game itself has hardcoded values to verify the string length, so the best way to replace levels is using a hack)`

Delete the last disclaimer, as it's fundamentally useless to provide in documentation. Try to explain what's in the parenthesis in the statement itself rather than putting these kinds of disclaimers in.